packages/js/src/ws/socket.ts

```diff
@@ -65,16 +65,25 @@ const mapToNotification = ({
     primaryAction: primaryCta && {
       label: primaryCta.content,
       isCompleted: actionType === ActionTypeEnum.PRIMARY && actionStatus === NotificationActionStatus.DONE,
+      redirect: {
+        target: primaryCta.target,
+        url: primaryCta?.url,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is there a specific reason for using optional chaining for `url` but not for `target`?‏

Reply:
No, missed it to update after the Redirect refactor